tests/testthat/test-ff_franchises.R

```diff
@@ -23,6 +23,6 @@ test_that("ff_franchises returns a tibble of franchises", {
   expect_tibble(dlf_franchises, nrows = 16)
   expect_tibble(jml_franchises, nrow = 12)
   expect_tibble(dlp_franchises, nrow = 12)
-  expect_tibble(joe_franchises, nrow = 12)
+  expect_tibble(joe_franchises, nrow = 16)
```

Comment:
This actually explains a pesky problem as to why the number of franchises had changed in this test: the league originally had 16 in 2020 but now has 12, and I couldn't figure out why this wasn't working before 😅  https://github.com/ffverse/ffscrapr/commit/1a47994d48e2159676162d8e79d2637bb7930a01